src/pages/ReportSettingsPage.js

```diff
@@ -144,7 +123,15 @@ class ReportSettingsPage extends Component {
                     onBackButtonPress={Navigation.goBack}
                     onCloseButtonPress={Navigation.dismissModal}
                 />
-                <ScrollView style={styles.flex1} contentContainerStyle={styles.p5} keyboardShouldPersistTaps="handled">
+                <Form
+                    formID={ONYXKEYS.FORMS.ROOM_SETTINGS_FORM}
+                    submitButtonText={this.props.translate('common.save')}
+                    style={[styles.mh5, styles.mt5, styles.flexGrow1]}
+                    validate={this.validate}
+                    onSubmit={this.validateAndUpdatePolicyRoomName}
```

Comment:
Let's update `validateAndUpdatePolicyRoomName` to `updatePolicyRoomName`.

Reply:
+1

Reply:
Fixed 💯